core/src/components/input/input.ionic.outline.scss

```diff
@@ -0,0 +1,162 @@
+@import "./input.vars";
+
+// Input Fill: Outline (Ionic Theme)
+// ----------------------------------------------------------------
+
+:host(.input-fill-outline) {
+  --border-radius: 8px;
+  --padding-start: 20px;
+  --padding-end: 20px;
+
+  min-height: 56px;
```

Comment:
We need to reach out to the designers for validation. Their [UX requirements](https://outsystemsrd.atlassian.net/wiki/spaces/UXD/pages/4011295407/Input+-+UX+Requirements#Requirements-List) has the default as 40px but their Figma shows 56px.

Reply:
Good catch; it turns out I based my styles off the extra-large size in the Figma by accident 😅 I'll get this updated to use the values for the default size.

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/29268/commits/b31f71929b3289154020e3bb44df0d425ce87e41, screenshots updated 👍